package.json

```diff
@@ -119,7 +119,12 @@
   },
   "dependencies": {
     "@paulcbetts/system-idle-time": "^1.0.4",
+    "ajv": "^6.2.1",
```

Comment:
Do we need this? I do not see it used anywhere.

Reply:
Yes we can remove it. But when we start our app with npm start, there are some warnings about dependencies being not resolved. I forcefully installed all the dependencies. I think its one one of the peerDependencies that needs to be added.

Reply:
What is the error? I am not getting any.

Reply:
Does it cause some errors in app? if not, and as it is only WARN, I would ignore it. I'd rather have less dependencies.